content/contributing/development/ui/scss_optimization.rst

```diff
@@ -0,0 +1,202 @@
+=========
+SCSS Tips
+=========
+
+There are many ways to lean and simplify SCSS. The first step is to establish if custom code is needed at all.
+
+Odoo's webclient has been designed to be modular, meaning that (potentially all) classes can be shared across views.
+Check the code before creating a new class. Chances are that there is already a class or a tag doing exactly what you're looking for.
+
+
+On top of that, Odoo relies on `Bootstrap <https://getbootstrap.com/docs/5.1/getting-started/introduction/>`_ (BS), one of the most complete CSS framework available. The framework has been customized in order to match Odoo design (both community and enterprise versions), meaning that you can use any BS class, directly in Odoo and achieve a visual result consistent with our UI.
+
+.. warning::
+   The fact that a class achieve the desired visual result doesn't necessarily mean that's the right one for the job. Be aware of classes triggering JS behaviors, for example.
+
+.. warning::
+   Take care about classes semantic. To apply a **button class** to a **title** is not just semantically wrong, it may also lead to migration issues and visual inconsistencies.
+
+
+The following sections describe tips to strip-down SCSS lines **when custom-code is the only way to go**.
+
+Browsers defaults
+=================
+
+By default, each browser renders content using an *user agent stylesheet*.
+To overcome inconsistencies, some of these rules are overridden by `Bootstrap Reboot <https://getbootstrap.com/docs/5.1/content/reboot/>`_.
+
+At this stage all "browser-specific-decoration" rules have been stripped away, but a big chunk of rules defining basic layout information are maintained (or reinforced by *Reboot* for consistency reason).
+
+You can rely on these rules.
+
+For example, apply `display: block;` to a `<div/>` is normally not necessary.
+
+.. code-block:: css
+
+   div.myElement {
+      display: block;
+      /* not needed 99% of the time */
+   }
+
+For the same reason, sometimes you may opt to switch html tag rather than adding a new CSS rule.
+
+.. code-block:: css
+
+   span.myElement {
+      display: block;
+      /* replace <span> with <div> instead
+         to get 'display: block' by default */
+   }
+
+
+Follows a non-comprehensive list of default rules:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Tag / Attribute
+     - Defaults
+   * - | `<div/>`, `<section/>`,
+       | `<header/>`, `<footer/>` ...
```

Comment:
Why do you force a line break here?

Reply:
To visually align elements and avoid the table to break the design